docs/platform_ccl.rst

```diff
@@ -0,0 +1,297 @@
+CC-Light Plaform Configuration
+------------------------------
+
+`hardware_configuration_cc_light.json <https://github.com/QE-Lab/OpenQL/blob/develop/tests/hardware_config_cc_light.json>`_ available inside the ``tests`` direcotry is an example configuration file for CC-Light platform. 
+
+``cc_light_compiler`` is the backend compiler used for CC-Light platform as specified by ``eqasm_compiler`` field as:
+
+.. code::
+
+    "eqasm_compiler" : "cc_light_compiler",
+
+Next section is ``hardware_settings`` which is used to configure various hardware settings of the platform as shown below:
+
+.. code-block::
+   :linenos:
+
+	"hardware_settings":
+	{
+		"qubit_number": 7,
+		"cycle_time" : 20,
+
+		"mw_mw_buffer": 0,
+		"mw_flux_buffer": 0,
+		"mw_readout_buffer": 0,
+		"flux_mw_buffer": 0,
+		"flux_flux_buffer": 0,
+		"flux_readout_buffer": 0,
+		"readout_mw_buffer": 0,
+		"readout_flux_buffer": 0,
+		"readout_readout_buffer": 0
+	}
+
+``qubit_number`` indicates the number of qubits available in the platform. Using qubits more than this number will raise an error. 
+
+``cycle_time`` is the value of the clock cycle time.
+
+Rest of the entries from line 6 to 14 specify various buffer times to be inserted in various operations due to control electronics setup. For example, ``mw_mw_buffer`` can be used to specify time to be inserted between a microwave operation followed by another microwave operation.
+
+.. note:: 
+	It is important to use same units to specify time in the configuraton
+	file. OpenQL uses ``cycle_time`` to convert all the times to cycles. 
+	These cycles are then used internally to schedule various operations.
+
+
+``resources`` section is used to specify/configure various resources
```

Comment:
Again, it is worth specifying what this information will constraint. Will it affect the mapping, the scheduling, etc?

Reply:
updated